pyiron_atomistics/vasp/base.py

```diff
@@ -1856,6 +1747,127 @@ def list_potentials(self):
         """
         return self.potential_list
 
+    def _set_kpoints(
```

Comment:
I am just reordering the functions to have the private functions at the end of the class definition.